Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/RagdollNodeWidget.cpp

```diff
@@ -154,6 +157,24 @@ namespace EMotionFX
                 m_ragdollNodeCard->show();
                 m_jointLimitWidget->show();
                 m_collidersWidget->show();
+
+                RagdollManipulatorData ragdollManipulatorData;
+                auto* actorInstance = GetActorInstance();
+                auto* selectedNode = GetNode();
```

Comment:
ActorInstance*? Node* is even the same amount of characters as auto, so auto is only hiding things here.